src/Core/Domain/CartRule/Exception/CannotDeleteCartRuleException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
```

Comment:
Missing license header 🙂

Reply:
And maybe add `declare(strict_types=1);`

Reply:
strict types is useless here as its empty class. (unless we require it in all files by linter)